velato/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "velato"
+version = "0.1.0"
+edition = "2021"
```

Comment:
This should use the same `workspace.* = true` as the other crates.

Reply:
Will do, thanks.